.circleci/config.yml

```diff
@@ -136,6 +143,14 @@ commands:
           command: cargo bench --all
 
 jobs:
+  Check Swift formatting:
+    macos:
+      xcode: "10.1.0"
+    steps:
+      - checkout
+      - run: brew install swiftlint swiftformat
+      - run: swiftlint --strict
+      - run: swiftformat --lint --swiftversion 4 --verbose megazords
```

Comment:
I am not sure how formatting on the C.I. does anything, I am probably misunderstanding this step.

Reply:
oh nvm, --lint checks to see if swiftformat was run on the code and reports discrepancies.